tests/service/servables/srl_test.py

```diff
@@ -16,4 +16,10 @@ def test_uses_named_inputs(self):
         result = model.predict_json(inputs)
 
         # TODO(joelgrus): update this when you figure out the result format
-        assert result
+        verbs = result.get("verbs")
+
+        assert verbs  # is not None
```

Comment:
somehow I had convinced myself that pylint didn't like that, but it seems ok with it, so I guess I was thinking of something else